### PR TITLE
Remove unnecessary numpy import

### LINKED ISSUE #9726: ModuleNotFoundError for Numpy when pytest.approx fails
### Description of the bug:
When using `pytest.approx` to compare lists in a test, a `ModuleNotFoundError` is raised for Numpy whenever the test fails. If the test succeeds, there is no such error.

It appears that `pytest.approx` does _not_ require Numpy to run, but the error message requires Numpy to display the details. Since `pytest.approx` does not require Numpy to function correctly, it should not require Numpy to display the error.

### Output of `pip list` from the virtual environment:
```
Package    Version
---------- -------
attrs      21.4.0
iniconfig  1.1.1
packaging  21.3
pip        22.0.3
pluggy     1.0.0
py         1.11.0
pyparsing  3.0.7
pytest     7.0.1
setuptools 49.2.1
tomli      2.0.1
```

### pytest and operating system versions:
```
$ python --version
Python 3.9.0
$ python -m pytest --version
pytest 7.0.1
```
macOS Big Sur

Version 11.6.2

### Minimal example:
```python
import pytest
def test_approx():
    assert [1, 2] == pytest.approx([1.001, 2.002])
```
#### Actual Result:
```
$ pytest
============================= test session starts ==============================
platform darwin -- Python 3.9.0, pytest-7.0.1, pluggy-1.0.0
rootdir: ****
collected 1 item                                                               

test_approx.py F                                                         [100%]

=================================== FAILURES ===================================
_________________________________ test_approx __________________________________

    def test_approx():
>       assert [1, 2] == pytest.approx([1.001, 2.002])
E       AssertionError: assert [1, 2] == approx([1.001...02 ± 2.0e-06])
E         (pytest_assertion plugin: representation of details failed: /Users/adalessa/Downloads/diffusion-master 2/venv/lib/python3.9/site-packages/_pytest/python_api.py:323: ModuleNotFoundError: No module named 'numpy'.
E          Probably an object has a faulty __repr__.)

test_approx.py:5: AssertionError
=========================== short test summary info ============================
FAILED test_approx.py::test_approx - AssertionError: assert [1, 2] == approx(...
============================== 1 failed in 0.04s ===============================
```
#### Expected result:
No `ModuleNotFoundError: No module named 'numpy'.` which makes the whole error message confusing and leads you to believe it failed because Numpy is not installed instead of the fact it was an assertion error.

### PR DESCRIPTION
**Pull request by kianelbo**
Closes #9726

### CODE REVIEW

File: changelog/9742.improvement.rst [hunk 1]
```diff
@@ -1 +0,0 @@
-Display assertion message without escaped newline characters with ``-vv``.
```
Comment (nicoddemus) [inline changelog/9742.improvement.rst:1]: This seems to have been removed by accident?

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Thanks @kianelbo for the PR!

Please take a look at my comment. 👍

File: changelog/9742.improvement.rst (hunk 1 above)
Comment (kianelbo) [reply changelog/9742.improvement.rst]: Sorry, I thought that file is dropped there as an example. I'll bring it back.

**Review by nicoddemus — APPROVED** [review]
Awesome, thanks!

**Pull request merged**